lib/sqlalchemy/util/langhelpers.py

```diff
@@ -1967,13 +1967,7 @@ class TypingOnly:
     def __init_subclass__(cls) -> None:
         if TypingOnly in cls.__bases__:
             remaining = set(cls.__dict__).difference(
-                {
-                    "__module__",
-                    "__doc__",
-                    "__slots__",
-                    "__orig_bases__",
-                    "__annotations__",
-                }
+                {name for name in cls.__dict__ if re.match("^__.+__$", name)}
```

Comment:
**Michael Bayer** (zzzeek) wrote:

let me just update this to be the one expression

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5287